index.html

```diff
@@ -6,19 +6,47 @@
     	<meta name="keywords" content="one,two,three">
 
 		<title>Blank Template</title>
+		<link rel="stylesheet" href="css/style.css" />
```

Comment:
don't forget to insert either a reset or normalize stylesheet before your `css/style.css` file